org.envirocar.app/res/layout/fragment_tracklist.xml

```diff
@@ -19,7 +19,9 @@
     with the enviroCar app. If not, see http://www.gnu.org/licenses/.
 
 -->
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+<androidx.coordinatorlayout.widget.CoordinatorLayout
```

Comment:
The reason here to use ``coordinatorlayout`` is that it doesn't let the snack bar, floating action bar and bottom navigation bar collide.
before:

![image](https://user-images.githubusercontent.com/75211982/145835069-0f6d70ec-a4d1-4493-ad50-d24cd259a724.png)

after:

![image](https://user-images.githubusercontent.com/75211982/145835228-06f0acc6-0cb8-4c6d-84fb-2fe77c0bb6c0.png)